lib/ffmpeg/qsv/encoder.py

```diff
@@ -171,6 +179,17 @@ def check_output(self):
       m = re.search(r"Using the VBR with lookahead \(LA\) ratecontrol method", self.output, re.MULTILINE)
       assert m is not None, "It appears that the lookahead did not load"
 
+    if vars(self).get("forced_idr", None) is not None:
+      iopts = "-i {encoded} -c:v copy".format(**vars(self))
+      oopts = (
+        "-vframes {frames} -bsf:v trace_headers"
+        " -f null - 2>&1 | grep 'nal_unit_type.*5'"
+        " | wc -l".format(**vars(self)))
+      self.copy_output = call(
+        "ffmpeg -v verbose {iopts} {oopts}".format(iopts = iopts, oopts = oopts))
+      m = re.search(str(self.frames), self.copy_output, re.MULTILINE)
+      assert m is not None, "It appears that the forced_idr did not work"
```

Comment:
Interesting approach... avoids excessive logging messages in original encode command 😉